README.md

```diff
@@ -74,7 +75,32 @@ documentation for more details.
   Environment
   Bootstrapping](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.pipelines-readme.html#cdk-environment-bootstrapping)
   for details.
-  
+
+## Developer Workflow
+
+Assuming you have your CDK app checked out on your local machine, here are the suggested steps
+to develop your GitHub Workflow.
+
+* Set up AWS Credentials your local environment. See [AWS Credentials](#aws-credentials) for the two
+  ways you may set up your credentials.
+
+* When you've updated your pipeline and are ready to deploy, run `cdk synth`. `cdk synth` will generate
+  a cloud assembly under the `cdk.out` folder and also the workflow file in `.github/workflows/deploy.yml`.
+  You should commit your code changes to GitHub along with `deploy.yml`, but you do not need to commit
+  the contents in `cdk.out`.
```

Comment:
It sounds more like you're explaining "how it works" than "what a user should do".

It would put those in different sections. "What it looks like to you" is more important for users than "how it works."

![image](https://user-images.githubusercontent.com/524162/157090967-3f0f1ff6-b33e-4953-965d-3d54269036e3.png)

Reply:
I will split the contents apart, though

Reply:
i couldn't think of a reasonable way to explain how it works, so I omitted it for now. perhaps you are right and it is unnecessary.